package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "tiptap-shadcn-vue",
   "type": "module",
-  "version": "0.1.0",
+  "version": "0.2.0",
```

```suggestion
  "version": "0.1.0",
```

`pnpm release` he'll change it himself.

---

Version 0.2.0 has already been published: https://www.npmjs.com/package/tiptap-shadcn-vue?activeTab=versions . After we make the necessary changes we can bump the version to 0.2.1.